tools/sof_ri_info/sof_ri_info.py

```diff
@@ -86,7 +119,10 @@
                             0x5, 0x76, 0x1b, 0x44, 0x55, 0x75, 0xd9, 0x32,
                             0x35, 0xf1, 0xec, 0x4d, 0x93, 0x73, 0xe6, 0xc4])
 
-KNOWN_KEYS = {CNL_INTEL_PROD_KEY : 'CNL Intel prod key',
+
+
```

Comment:
Do you really need additional 2 empty lines here?

Reply:
ack. I've removed 2 extra lines.